### ISSUE TITLE
[Bug][a11y] Can't select separately the number of selected tabs to be added to a collection on Android 5 & 6 devices

### ISSUE THREAD

**Issue by AndiAJ** [mozilla-mobile/fenix#16474]
## Prerequisites
Talkback should be enabled
Have a couple couple of open tabs

## Steps to reproduce
1. Tap the "Save to collection" button
2. Select a couple of tabs
3. Tap on the number of selected tabs from the action bar

### Expected behavior
Should be able to select only the number of selected tabs item

### Actual behavior
Tapping the the number of selected tabs actually selects the entire view
It gets skipped when I try to reach it with the swipe gesture

### Device information

* Android device: 
• OnePlus A3 (Android 6.0.1)
• LG Nexus 4 (Android 5.1.1)

* Fenix version: Nightly Build 201110

### Notes
Noticed while checking #16426 
 
Works properly on:
✔️ Google Pixel 3a (Android 11)
✔️ Huawei Mate 20 Lite (Android 10)
✔️ OnePlus 6T (Android 9)
✔️ Nexus 5 (Android 6.0.1)
